com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+## [1.5.2] - 2023-07-11
```

Comment:
Could you the new change set section to:
## [Unreleased]
_(and exclude the date)_

I am pretty sure we will be doing a patch, but we typically just mark it as "Unreleased" until we know for sure what the version and date will be.

Reply:
done! :)